src/transactions/TransactionFrame.cpp

```diff
@@ -286,6 +286,46 @@ TransactionFrame::processSeqNum(LedgerManager& lm, LedgerDelta& delta)
     }
 }
 
+bool
+TransactionFrame::processSignatures(SignatureChecker& signatureChecker,
+                                    Application& app, LedgerDelta& delta)
+{
+    if (app.getLedgerManager().getCurrentLedgerVersion() < 10)
+    {
+        return true;
+    }
+
+    auto allOpsValid = std::all_of(
+        std::begin(mOperations), std::end(mOperations),
+        [&signatureChecker, &app](auto const& op) {
+            return op->checkSignature(signatureChecker, app, nullptr);
+        });
+
+    removeUsedOneTimeSignerKeys(signatureChecker, delta,
```

Comment:
another problem that @jonjove found: `removeUsedOneTimeSignerKeys` must use the `delta` version of `LoadAccount` (master is buggy as well but bug has more potential implications with this change)

Reply:
This should be changed for all versions or only for versions >= 10? I'm not sure if this has any impact on generated buckets?